grapher/rule.go

```diff
@@ -58,7 +59,13 @@ func (r *GraphUnitRule) Target() string {
 
 func (r *GraphUnitRule) Prereqs() []string {
 	ps := []string{filepath.ToSlash(filepath.Join(r.dataDir, plan.SourceUnitDataFilename(unit.SourceUnit{}, r.Unit)))}
-	ps = append(ps, r.Unit.Files...)
+	for _, file := range r.Unit.Files {
+		if _, err := os.Stat(file); err != nil && os.IsNotExist(err) {
```

Comment:
@beyang made a suggestion to use just `os.IsNotExist(err)` (i.e., drop the `err != nil`) in another PR with similar code, but not here.

Is there a reason for that? Which style was deemed to be preferable?